op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -27,8 +30,27 @@ func TestExecFromSameAddressInALoop(gt *testing.T) {
 	}
 	t := devtest.SerialT(gt)
 	sys := presets.NewSimpleInterop(t)
-	eoaA := sys.FunderA.NewFundedEOA(eth.MillionEther)
-	eoaB := sys.FunderB.NewFundedEOA(eth.MillionEther)
+
+	// Use proxyd if available.
+	l2ELA := sys.L2ELA
+	l2ELB := sys.L2ELB
+	if proxydsA := match.Proxyd.Match(sys.L2ChainA.Escape().L2ELNodes()); len(proxydsA) > 0 {
+		l2ELA = dsl.NewL2ELNode(proxydsA[0])
+	}
+	if proxydsB := match.Proxyd.Match(sys.L2ChainB.Escape().L2ELNodes()); len(proxydsB) > 0 {
+		l2ELB = dsl.NewL2ELNode(proxydsB[0])
+	}
```

Comment:
Later would be nice to integrate this logic into the orchestrator/preset; does not need to block.

Reply:
I agree it's a bit ugly. I didn't think of anything more appealing and would like to hear you're ideas later.